### ISSUE TITLE
'SLURMCluster' object has no attribute 'plan'

### ISSUE THREAD

**Issue by trey-stafford** [dask/dask-jobqueue#349]
Attempting to use the `adapt` method after creating a slurm cluster results in the following error:
```
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2b764c392ef0>>, <Task finished coro=<AdaptiveCore.adapt() done, defined at /projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/distributed/deploy/adaptive_core.py:170> exception=AttributeError("'SLURMCluster' object has no attribute 'plan'",)>)
Traceback (most recent call last):
  File "/projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/distributed/deploy/adaptive_core.py", line 182, in adapt
    recommendations = await self.recommendations(target)
  File "/projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/distributed/deploy/adaptive_core.py", line 133, in recommendations
    plan = self.plan
  File "/projects/trst2284/miniconda3/envs/lisa/lib/python3.6/site-packages/distributed/deploy/adaptive.py", line 101, in plan
    return self.cluster.plan
AttributeError: 'SLURMCluster' object has no attribute 'plan'
```

I am using dask-jobqueue v0.6.3:
```
$ conda list jobqueue
# packages in environment at /projects/trst2284/miniconda3/envs/lisa:
#
# Name                    Version                   Build  Channel
dask-jobqueue             0.6.3                      py_0    conda-forge/channel/main
```

Perhaps I am missing a step in setting up an adaptive cluster? Any advice would be appreciated!

**Comment by jhamman** [comment]
@trey-stafford - We released v0.7 yesterday. If its an option for you, this should be fixed in that release.

**Comment by trey-stafford** [comment]
@jhamman that does indeed fix the issue. Thanks!
